src/sites/amsterdam/components/contact/Contact.tsx

```diff
@@ -0,0 +1,64 @@
+import { Breadcrumb, Column, Fieldset, Grid, Heading, Link, Paragraph, Radio } from '@amsterdam/design-system-react'
+import { siteUrl } from '../../constants'
+import FormNavigation from '../shared/FormNavigation'
+import { FormEvent } from 'react'
+import { useNavigate } from 'react-router-dom'
+
+export const Contact = () => {
+  const navigate = useNavigate()
+
+  const handleSubmit = async (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault()
+
+    const formData = new FormData(event.currentTarget)
+    const soort = formData.get('soort')
+
+    if (typeof soort === 'string') {
+      navigate(`${siteUrl}contact/${soort}`)
+    }
+  }
+
+  return (
+    <Grid paddingBottom="medium">
+      <Grid.Cell span={{ narrow: 4, medium: 5, wide: 8 }} start={{ narrow: 1, medium: 1, wide: 2 }}>
+        <Breadcrumb>
+          <Breadcrumb.Item href={siteUrl}>Home</Breadcrumb.Item>
+          <Breadcrumb.Item href="#">Contact</Breadcrumb.Item>
```

Comment:
We only display parent pages in the breadcrumb, not the current one. There may be other occurrences.

Reply:
Done